build/npm/clarity-icons.json

```diff
@@ -10,7 +10,7 @@
         "url" : "ssh://git@git.eng.vmware.com/clarity.git"
     },
     "peerDependencies": {
-        "@webcomponents/custom-elements": "1.0.0"
+        "@webcomponents/custom-elements": "^1.0.2"
```

Comment:
This should be `^1.0.0` rather than `^1.0.2`.

Reply:
Good point.